src/client/src/orangehrmPerformancePlugin/components/AddTrackerLogModal.vue

```diff
@@ -0,0 +1,178 @@
+            icon-name="hand-thumbs-up-fill"
+            :class="{
+              'orangehrm-add-tracker-rating-button': true,
+              '--positive': true,
+              '--deselected': positiveSelected === false,
+            }"
+            @click="onClickPositive"
+          />
+          <oxd-button
+            display-type="text"
+            :label="$t('performance.negative')"
+            icon-name="hand-thumbs-down-fill"
+            :class="{
+              'orangehrm-add-tracker-rating-button': true,
+              '--negative': true,
+              '--deselected': negativeSelected === false,
+            }"
+            @click="onClickNegative"
+          />
+        </div>
+      </oxd-form-row>
+      <oxd-form-row>
+        <oxd-input-field
+          v-model="trackerLog.comment"
+          type="textarea"
+          :label="$t('general.comment')"
+          :placeholder="$t('general.type_here')"
+          :rules="rules.comment"
+        />
+      </oxd-form-row>
+      <oxd-divider />
+      <oxd-form-actions class="orangehrm-form-action">
+        <required-text />
+        <oxd-button
+          display-type="ghost"
+          :label="$t('general.cancel')"
+          @click="onCancel"
+        />
+        <oxd-button
+          display-type="secondary"
+          :label="$t('general.save')"
+          type="submit"
+        />
+      </oxd-form-actions>
+    </oxd-form>
+  </oxd-dialog>
+</template>
+
+<script>
+import Dialog from '@ohrm/oxd/core/components/Dialog/Dialog';
+import {
+  required,
+  shouldNotExceedCharLength,
+} from '@/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+
+const trackerLogModel = {
+  log: '',
+  comment: '',
+};
+
+export default {
+  name: 'AddTrackerLogModal',
+  components: {
+    'oxd-dialog': Dialog,
+  },
+  props: {
+    trackerId: {
+      type: Number,
+      required: true,
+    },
+  },
+  emits: ['close'],
+  setup(props) {
+    // TODO change to window.appGlobal.baseUrl
+    const http = new APIService(
+      'https://942be86c-56c6-42e3-ac85-874a20c7ce9b.mock.pstmn.io',
+      `api/v2/performance/trackers/${props.trackerId}/logs`,
+    );
+
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      positiveSelected: true,
+      negativeSelected: false,
+      trackerLog: {...trackerLogModel},
+      rules: {
+        log: [required, shouldNotExceedCharLength(150)],
+        comment: [required, shouldNotExceedCharLength(3000)],
+      },
+    };
+  },
+  methods: {
+    onClickPositive() {
+      this.positiveSelected = true;
+      this.negativeSelected = false;
+    },
+    onClickNegative() {
+      this.negativeSelected = true;
+      this.positiveSelected = false;
+    },
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .create({
+          log: this.trackerLog.log,
+          comment: this.trackerLog.comment,
+          achievement: this.positiveSelected ? 1 : 2,
```

Comment:
instead of achievement shall we go with a word like rating after @RajithaKumara 's approval as well?

Reply:
Since like you mentioned before negative achievement doesn't make sense neh?

Reply:
changed to `rating`. will keep this open for Rajitha's comment

Reply:
better to go with as `achievement`, since spec also refers as achievement